docs/maps/connect-to-ems.asciidoc

```diff
@@ -190,7 +191,11 @@ services:
 [[elastic-maps-server-data]]
 ==== Data
 
-{hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. A minimal basemap is provided with {hosted-ems}. This can be used for testing environments but won't be functional for normal operations. The full basemap (around 90GB file) needs to be mounted on the Docker container for {hosted-ems} to run normally.
+{hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. Basemaps up to zoom level 8 are bundled in the Docker image. These basemaps are sufficient for maps and dashboards at country level. But, if you want to present maps with higher detail, follow the instructions of the front-page to download and configure the appropriate basemaps database. The most detailed basemaps at zoom level 14 are good for street level maps but require ~90GB of disk space.
+
+
+[role="screenshot"]
```

Comment:
Can this image either show the whole screen, or be cut just to show the zoom level menu?

Reply:
I defer to Nick on `basemaps database`.

Reply:
I'll crop it to just the zoom level menu since we don't want to show the URL behind it :+1: